### PR TITLE
7025 circuit redundancy

### PR DESCRIPTION
**Pull request by arthanson**

### CODE REVIEW

File: netbox/templates/circuits/circuitgroupassignment.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <th scope="row">{% trans "Priority" %}</th>
+            <td>{{ object.priority }}</td>
+          </tr>
+        </table>
+      </div>
+      {% include 'inc/panels/tags.html' %}
+      {% plugin_left_page object %}
+    </div>
+    <div class="col col-md-6">
+      {% include 'inc/panels/related_objects.html' %}
```
Comment (jeremystretch) [inline netbox/templates/circuits/circuitgroupassignment.html:37]: This panel is not needed.

Comment (jeremystretch) [inline netbox/templates/circuits/circuitgroupassignment.html:37]: Missing the custom fields panel.

**Review by jeremystretch — CHANGES_REQUESTED** [review]
Some other notes:

- The CircuitGroup view should include a "circuits" tab to easily list assigned circuits.

- There should be some convenient path to assign a circuit to a group from the circuit view.

- Looks like we're missing a [search index](https://netbox.readthedocs.io/en/stable/development/adding-models/#9-create-a-searchindex-subclass) for CircuitGroup.

File: netbox/circuits/models/circuits.py [hunk 2]
```diff
@@ -151,6 +153,71 @@ def clean(self):
...
+
+    def __str__(self):
+        return self.name
+
+    def get_absolute_url(self):
+        return reverse('circuits:circuitgroup', args=[self.pk])
+
+
+class CircuitGroupAssignment(CustomFieldsMixin, ExportTemplatesMixin, TagsMixin, ChangeLoggedModel):
+    circuit = models.ForeignKey(
```
[the remark this answers is not in the document]
Comment (arthanson) [reply netbox/circuits/models/circuits.py:184]: I looked at it but it is never used anywhere and adds complexity to the related_counts.

[the remark this answers is not in the document]
Comment (arthanson) [reply netbox/circuits/models/circuits.py:184]: RE "The CircuitGroup view should include a "circuits" tab to easily list assigned circuits.

There should be some convenient path to assign a circuit to a group from the circuit view.
"
For the detail view the "Related Objects" I thought took the place of that, that is the way all the other group ones do it from what I can see. 

Also, I don't see a paradigm for the other group objects having an assignment on the object page (Tunnel Group, Tenant Group)?  Do we have a UI pattern for this somewhere?

File: netbox/templates/circuits/circuitgroupassignment.html (hunk 1 above)
Comment (jeremystretch) [reply netbox/templates/circuits/circuitgroupassignment.html]: Disregard; we'll keep this in place instead of adding a separate tab view.

**Pull request merged**